package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "eslint": "^4.19.1",
     "eslint-plugin-react": "^7.7.0",
+    "gatsby-new-blog": "^1.0.4",
```

Comment:
Why do we need this?

Reply:
Oops..was adding a package to automate the process of writing a new blog post. Removing it for now.